src/components/KMultiselect/KMultiselect.vue

```diff
@@ -211,7 +215,7 @@
           v-for="item, idx in visibleSelectedItemsStaging"
           :key="`${item.key ? item.key : idx}-badge`"
           class="mr-1 mt-2"
-          dismissable
+          :dismissable="!(item.selected && item.disabled)"
```

Comment:
nit(non-blocking): this seems more readable IMO
```suggestion
          :dismissable="item.selected && !item.disabled"
```